dev-python/uri_template/uri_template-1.1.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_HASH="8547d60a57dd8aabffa7063fbc0b120c9902a130"
+
+PYTHON_COMPAT=( pypy3 python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="URI Template expansion in strict adherence to RFC 6570"
+HOMEPAGE="https://gitlab.linss.com/open-source/uri_template/"
+SRC_URI="https://gitlab.linss.com/open-source/${PN}/-/archive/v${PV}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="test"
```

Comment:
We should be able to drop this, since we don't need extra deps to run the tests.